android/app/src/main/kotlin/com/jabook/app/jabook/audio/MediaSourceValidator.kt

```
@@ -0,0 +1,197 @@
+        }
+
+        // WAV: "RIFF" at 0-3 and "WAVE" at 8-11
+        if (header.size >= 12 &&
+            header[0] == 'R'.code.toByte() &&
+            header[1] == 'I'.code.toByte() &&
+            header[2] == 'F'.code.toByte() &&
+            header[3] == 'F'.code.toByte() &&
+            header[8] == 'W'.code.toByte() &&
+            header[9] == 'A'.code.toByte() &&
+            header[10] == 'V'.code.toByte() &&
+            header[11] == 'E'.code.toByte()
```

detekt (1.23.8)

[warning] This condition is too complex (9). Defined complexity threshold for conditions is set to '4'

(detekt.complexity.ComplexCondition)